README.md

```diff
@@ -31,56 +28,76 @@ Membership to this organization requires that the student be enrolled at Clemson
 
 Officer positions are as follows:
 
 * President
 * Vice President
-* System Architect
 * Treasurer
 * Secretary
-* Webmaster
+* Competition and Events Manager
+* Training Manager
+
+Should an officer be unable to fulfill their duties due to health issues, they are relieved of their position, and their position is to be filled at the next meeting by a snap election.
 
 ## Article VI
 
+### Appointed Positions (Titles, Term of Office, How Selected)
+
+Each appointed position, with the exception of Lab Administrator, serves at-will of the board of Officers, is appointed by 2/3 majority of Officers' approval, and is relieved via graduation or 2/3 majority of sitting Officers' approval. The position of Lab Administrator will be filled by a unanimous vote of the current officers and advisors, relieved only via graduation or unanimous vote of current officers and advisors.
+
+Appointed positions are as follows:
+
+* Lab Administrator
+* Website Administrator
+* Social Event Manager
+* CyberRange Project Manager
+* Weekly Meeting Manager
+* Blue Team Trainer
+* Red Team Trainer
+
+Should an appointee be unable to fulfill their duties due to health issues, they are relieved of their duties, and their position is to remain vacant until another is appointed by the Officers.
+
+## Article VII
+
 ### Advisor (Term of Service, How Selected)
 
 Advisors will be selected by a unanimous vote of the current officers. Advisors can retain the position until voluntary abdication or until a recall referendum.
 
-## Article VII
+## Article VIII
 
 ### Meetings (Frequency, Special Meetings and Who Calls Them)
 
 Meetings will occur no less than once a month and these monthly meetings must meet for at least one hour. Special meetings can be called by any officer, advisor or at a majority vote of the current members.
 
-## Article VIII
+## Article IX
 
 ### Quorum (Number of Members Required to Transact Business)
 
 At least 4 members must be present to conduct business and this must include at least one officer or advisor. Any votes or other acts of business must be recorded in a format that can be provided to any officer or advisor by request.
 
 Any issues dealing with organizational funds must be approved by a majority vote of members with at least one officer or advisor present or by a majority vote of officers and advisors where all officers and at least one advisor are present.
 
-## Article IX
+## Article X
 
 ### Referendum and Recall (Procedures and Handling)
 
 Any officer or advisor can be removed from their position by a majority vote of all active members and officers (excluding the officer being recalled).
```

Comment:
Should we add appointed positions here?

Reply:
According to @houstdav000, appointed positions are purely functionary and have no voting rights.  It seems odd to add them to the quorum requirements or subject them to recall provisions outside the provisions held by the officer corps.